aiosmtpd/tests/test_handlers.py

```diff
@@ -1,9 +1,11 @@
+import asyncio
```

Comment:
Why? Now imports sorted in alphabetically, what the point to break it if move `asyncio` under `sys`?